mediawiki123-apb/roles/provision-mediawiki123-apb/tasks/main.yml

```diff
@@ -55,16 +55,6 @@
         value: postgres
       - name: MEDIAWIKI_SITE_SERVER
         value: '{{ route.route.spec.host }}'
-      - name: POSTGRESQL_HOST
-        value: '{{ postgresql_host }}'
-      - name: POSTGRESQL_DATABASE
-        value: '{{ postgresql_database }}'
-      - name: POSTGRESQL_PORT
-        value: '{{ postgresql_port }}'
-      - name: POSTGRESQL_USER
-        value: '{{ postgresql_user }}'
-      - name: POSTGRESQL_PASSWORD
-        value: '{{ postgresql_password }}'
       image: docker.io/jmontleon/mediawiki123:latest
```

Comment:
In my opinion, we should keep all the images that are working in one place so we can formally tag those images with releases of the broker and our tooling. I think it makes sense for testing to use your own repo. But for community consumption, I think all our images should be in the same place.  So if this image is 'stable' for the most part, I think it's worth to publish it to the ansibleplaybookbundle repo. What do you think @jmontleon?